docs/404.html

```diff
@@ -2,6 +2,15 @@
 layout: default
 ---
 
+<script>
+  window.onload = function () {
+      // Redirect to the real documentation site if github pages url is attempted. 
+      if (window.location.href.includes("forcedotcom.github.io/salesforcedx-vscode")) {
+          window.location.href = "https://developer.salesforce.com/tools/vscode";
+      }
+  }
+</script>
```

Comment:
I looked around a bit to find the right solution here.  This is the cleanest option I could find.  I also considered having the redirect be baked into Jeckll or using the settings for the repo to do the redirect, but neither turned out to be very straightforward.